docs/widget.rst

```diff
@@ -8,7 +8,7 @@ It embeds *Leaflet.draw* in version *0.4.0*.
 .. image :: https://f.cloud.github.com/assets/546692/1048836/78b6ad94-1094-11e3-86d8-c3e88626a31d.png
 
 
-In Adminsite
+In admin.py
 ------------
```

Comment:
Please match the number of `-` under the title.

Reply:
Sorry, I don't understand and don't know RST. Are you saying the "-" line should be the same length and the line above it?

Reply:
Yes. cf http://docutils.sourceforge.net/docs/ref/rst/restructuredtext.html#sections

Reply:
it doesn't really matter, as long as it's equal or more.  it works this way too.